Packs/GoogleChronicleBackstory/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Chronicle",
     "description": "Use the Chronicle integration to retrieve Asset alerts or IOC Domain matches as Incidents. Use it to fetch a list of infected assets based on the indicator accessed. This integration also provides reputation and threat enrichment of indicators observed in the enterprise",
     "support": "partner",
-    "currentVersion": "1.1.4",
+    "currentVersion": "2.0.0",
     "certification": "certified",
     "author": "Chronicle",
     "url": "",
```

Comment:
I suggest adding a URL here

Reply:
@guykeller We tried adding the URL, but we do not have the access to edit the files here. Can you please add this URL `https://go.chronicle.security/contact` for us?

Reply:
> I suggest adding a URL here

@guykeller We tried adding the URL, but we do not have the access to edit the files here. Can you please add this URL https://go.chronicle.security/contact for us?

Reply:
Sure!